tests/cases/fourslash/renameExportSpecifier.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: a.ts
+////const name = {};
+////export { name as name/**/ };
```

Comment:
Can you add baseline for rename location of  `export { name }`  or convert existing test to baseline for maintainability